cleverpush/src/main/java/com/cleverpush/service/NotificationService.java

```diff
@@ -572,4 +598,21 @@ private static Bitmap loadImageFromDisc(Context context, String fileName) {
 
     return bitmap;
   }
+
+  private PendingIntent createActionPendingIntent(Context context, NotificationAction action, Notification notification, String subscriptionStr, int actionIndex, int requestId) {
+    Intent actionIntent = getTargetIntent(context);
+    Notification actionNotification = notification;
+    if (action.getUrl() != null && !action.getUrl().isEmpty()) {
+      actionNotification.setUrl(action.getUrl());
+    }
+    String notificationStr = new Gson().toJson(actionNotification);
+    actionIntent.putExtra("actionIndex", String.valueOf(actionIndex));
+    actionIntent.putExtra("notificationId", requestId);
```

Comment:
rename to requestId?

Reply:
As you can see NotificationManagerCompat's cancel method declaration. They have mentioned the ID of the notification in the param that's why I used notificationId. 

Should I change the name?

<img width="769" alt="Screenshot 2023-12-21 at 10 52 58 AM" src="https://github.com/cleverpush/cleverpush-android-sdk/assets/129381063/1e08b7b6-3321-43f7-914f-08c92f193d13">